src/org/thoughtcrime/securesms/connect/AccountManager.java

```diff
@@ -302,15 +172,14 @@ public void showSwitchAccountMenu(Activity activity) {
                 .setSingleChoiceItems(menu.toArray(new String[menu.size()]), presel, (dialog, which) -> {
```

Comment:
```suggestion
                .setSingleChoiceItems(menu.toArray(new String[0]), presel, (dialog, which) -> {
```

See https://stackoverflow.com/questions/53284214/toarray-with-pre-sized-array

Reply:
that looks weird to me ... but well, java is weird.